composer.json

```diff
@@ -34,6 +34,7 @@
         "doctrine/orm": "^2.5",
         "doctrine/annotations": "^1.2",
         "friendsofsymfony/user-bundle": "^2.0@dev",
+        "guzzlehttp/guzzle": "^6.0",
```

Comment:
AFAIK it doesn't support async requests... and it has not been updated since 2015.